AzimAkmatov/Test-repo
# This is a branch which has ec2.yaml file for AWS CloudFormation

Created new file ec2.yaml for AWS CloudFormation. Created folder "mode" and file "hello.txt" in that folder.
How to test my code:
1. Open filex
2. Run "some command"
3. If you see "X" then its working
